### ISSUE TITLE
Paginator QueryBuilder not escaping rowcount column

### ISSUE THREAD

**Issue by dnapierata** [phalcon/cphalcon#12950]
### Expected and Actual Behavior

When using the paginator with a `GROUP BY` and `HAVING` clause, the sql that is generated does not escape the aliased column and thus I am receiving a syntax error from my database (MSSQL 12.0). [https://github.com/phalcon/cphalcon/blob/master/phalcon/paginator/adapter/querybuilder.zep#L228](https://github.com/phalcon/cphalcon/blob/master/phalcon/paginator/adapter/querybuilder.zep#L228)

I would expect Line 228 to be 
```zep
let row = db->fetchOne("SELECT COUNT(*) as \"rowcount\" FROM (" .  sql["sql"] . ") as T1", Db::FETCH_ASSOC, sql["bind"]),
```
### Details

* Phalcon version: (`3.2.0`)
* PHP Version: (` 7.0.18-0ubuntu0.16.04.1`)
* Operating System: Linux
* Installation type:  installing via package manager
* Server: Apache
* Other related info (Database, table schema): MSSQL 12.0

**Comment by sergeyklay** [comment]
Phalcon does not have build-in MSSQL dialect class. So it is most likely a new feature request for Incubator.

**Comment by dnapierata** [comment]
I realize MSSQL isn't currently supported by phalcon but this fix is just to make the generated SQL more standards compliant and in fact it is already being done here [https://github.com/phalcon/cphalcon/blob/master/phalcon/mvc/model.zep#L1135](https://github.com/phalcon/cphalcon/blob/master/phalcon/mvc/model.zep#L1135)

I already have the dialect and adapter classes for sql server but this does not solve the issue I have posted above. I am just asking you add double quotes around `rowcount` so that the query builder paginator is more compatible with future db dialects.

**Comment by Jurigag** [comment]
Can you do PR with fix to 3.2.x branch?

**Comment by sergeyklay** [comment]
Fixed in the `3.2.x branch`. Feel free to open a new issue if the problem appears again. Thank you for contributing.
